src/webui/webapplication.cpp

```diff
@@ -466,8 +466,7 @@ Http::Response WebApplication::processRequest(const Http::Request &request, cons
     }
     catch (const HTTPError &error) {
         status(error.statusCode(), error.statusText());
-        if (!error.message().isEmpty())
-            print(error.message(), Http::CONTENT_TYPE_TXT);
+        print(!error.message().isEmpty() ? error.message() : error.statusText(), Http::CONTENT_TYPE_TXT);
```

Comment:
I think you should still add parentheses to the first parameter of the `print` function.

Reply:
We really need to automate the `uncrustify` script as a GitHub Action. It's not worth either of our time dealing with small formatting issues like this.

Reply:
I agree in general. However the last time I checked none of the formatting tools are able to comply to qbt coding style. :\

Reply:
I think that may say something about our chosen coding style 😁

Reply:
>I think that may say something about our chosen coding style 

I think that may say something about quality of all such "formatting tools" as well.